examples/cliexample/main.go

```diff
@@ -45,7 +45,7 @@ func main() {
 		}
 	}()
 
-	engine, err := ewf.NewEngine(store)
+	engine, err := ewf.NewEngine(ewf.Withstore(store))
```

Comment:
WithStore*(

Reply:
probably I was smoking something :D This should be on the Engine when I think about it now.